tsconfig.json

```diff
@@ -13,5 +13,5 @@
     "sourceMap": true,
     "skipLibCheck": true
   },
-  "exclude": ["node_modules", "**/*.spec.ts"]
+  "exclude": ["node_modules"]
```

Comment:
nit: `node_modules` is excluded by default.
https://www.typescriptlang.org/tsconfig#exclude